ui/src/actions/draftItem.js

```diff
@@ -244,15 +244,12 @@ export function postCreateDraft(data = {}, ana_type) {
       })
       .catch(error => {
         dispatch(createDraftError(error.response));
-        cogoToast.error(
-          "There is an error, please make sure you are connected and try again",
-          {
-            position: "top-center",
-            heading: error.message,
-            bar: { size: "0" },
-            hideAfter: 3
-          }
-        );
+        cogoToast.warn("Make sure you selected a type for your analysis", {
```

Comment:
Maybe we should display the correct error message from response? 

PS This goes together with the other comment, about making "form validation" before request is sent

Reply:
Message received properly from `error.response.data`